tools/io_tracer_parser_tool.cc

```diff
@@ -0,0 +1,108 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//    This source code is licensed under both the GPLv2 (found in the
+//    COPYING file in the root directory) and Apache 2.0 License
+//    (found in the LICENSE.Apache file in the root directory).
+
+#ifndef ROCKSDB_LITE
+
+#include "rocksdb/io_tracer_parser_tool.h"
+
+#include <cinttypes>
+#include <cstdio>
+#include <memory>
+
+#include "port/lang.h"
+#include "rocksdb/env.h"
+#include "trace_replay/io_tracer.h"
+
+namespace ROCKSDB_NAMESPACE {
+
+IOTraceRecordParser::IOTraceRecordParser(std::string input_file)
+    : input_file_(input_file) {}
+
+void IOTraceRecordParser::PrintHumanReadableIOTraceRecord(
+    const IOTraceRecord& record) {
+  fprintf(stdout,
+          "Access Time : %" PRIu64 ", File Operation: %s, Latency: %" PRIu64
```

Comment:
Should we give an option to decide whether the output file is for human read or for program processing? If we out put a file in this format, might not be convenient for other applications to process.

Reply:
This is for human read. I am working on providing an output_file option. I am not sure about the program processing.